libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/auth/HttpAccessTest.java

```diff
@@ -48,19 +53,22 @@
 public class HttpAccessTest extends InstrumentationTestCase {
```

Comment:
Since HttpAccess is no longer used to do get/post etc, the value of this test suite is very limited.